portals/publisher/src/main/webapp/site/public/locales/en.json

```diff
@@ -216,8 +216,8 @@
   "Apis.Create.streaming.Components.SelectPolicies.business.plans": "Protocol",
   "Apis.Create.streaming.Components.create.and.publish.btn": "Create & Publish",
   "Apis.Create.streaming.Components.create.btn": "Create",
-  "Apis.Details.\n                                                                                            Environments.Environments.\n                                                                                            pending.chip": "Pending",
-  "Apis.Details.\n                                                                                Environments.deploy.vhost": "VHost",
+  "Apis.Details.\r\n                                                                                            Environments.Environments.\r\n                                                                                            pending.chip": "Pending",
```

Comment:
@Ratheshan03 The changes to this `en.json` file will not be needed as the default messages will not change with your fix.

Reply:
Sure I'll revoke the changes i made in the json file.

Reply:
@tharikaGitHub Thanks for the feedback! I've removed the `en.json` changes in the latest commit. The PR now only contains the UI fix changes to `OperationGovernance.jsx`.